wrapanapi/systems/container/podman.py

```diff
@@ -0,0 +1,110 @@
+from abc import ABCMeta
+from datetime import datetime
+
+from podman import PodmanClient
+from proto.utils import cached_property
+
+from wrapanapi.entities.base import Entity
+from wrapanapi.systems.base import System
+
+
+class PodmanContainer(Entity, metaclass=ABCMeta):
+    def __init__(self, system, raw=None, **kwargs):
+        """
+        Constructor for an PodmanContainer tied to a specific host.
+
+        Args:
+            system: a Podman system object
+            raw: Raw container object if already obtained, or None
+        """
+        self._name = kwargs.get("name")
+        self._id = kwargs.get("id")
+        if not self._name:
+            raise ValueError("missing required kwargs identifier: 'name'")
+
+        super().__init__(system, raw, **kwargs)
+        self._api = self.system.containers_collection
+
+    @property
+    def _identifying_attrs(self):
+        return {"name": self._name, "id": self._id}
+
+    @property
+    def name(self):
+        return self._name
+
+    @property
+    def id(self):
+        return self._id
+
+    @property
+    def uuid(self):
+        return self.id
+
+    @property
+    def creation_time(self):
+        return datetime.fromisoformat(self.raw.attrs["Created"])
+
+    def delete(self, force=False):
+        return self.raw.remove(force=force)
+
+    def stop(self):
+        return self.raw.stop()
+
+    def cleanup(self):
+        return self.delete()
+
+    def refresh(self):
+        container = self._api.get(self.name)
+        self.raw = container
+        return self.raw
+
+
+class Podman(System):
+    def __init__(
+        self, hostname, username, protocol="http+ssh", port=22, verify_ssl=False, **kwargs
+    ):
+        super().__init__(kwargs)
```

Comment:
does this need to be unpacked here?
```suggestion
        super().__init__(**kwargs)
```